.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,6 @@ the requirements below.
 
 -->
 
-##### Checklist
```

Comment:
We don't have any specific commit guidelines in this repo, other than the DCO bot enforcing the signoff.